### PR TITLE
Create mode.js

### PR DESCRIPTION
**Pull request by Mohit-Gaur**
Please check all that apply.

#### This PR is for:
- [x] Algorithm
- [ ] Cheatsheet
- [ ] Resource
- [ ] Other

#### Programming Language: JavaScript

- [x] Have you implemented it on your local machine or any online compiler?
- [x] There is the main( ) function to run the program and all the inputs which are being taken are clearly printed on the console or well commented in the file.
- [x] Filename is self-explanatory or there is Readme.md file explaining the necessary details of the algorithm.

### CODE REVIEW

**Review by Mohit-Gaur — COMMENTED** [review]
done

File: JavaScript/Mode/mode.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    a.forEach(e => {
+        if (!(e in count)) {
+            count[e] = 0;
+        }
+
+        count[e]++;
+    });
+
+    let bestElement;
+    let bestCount = 0;
```
[the remark this answers is not in the document]
Comment (Mohit-Gaur) [reply JavaScript/Mode/mode.js:13]: done

**Comment by Mohit-Gaur** [comment]
changes done

**Review by starkblaze01 — APPROVED** [review]
Both algorithms look great!
:tada: Thanks for your contribution.

**Pull request merged**